cmd/kubeadm/app/features/features.go

```diff
@@ -32,12 +32,15 @@ const (
 	IPv6DualStack = "IPv6DualStack"
 	// PublicKeysECDSA is expected to be alpha in v1.19
 	PublicKeysECDSA = "PublicKeysECDSA"
+	// RootlessControlPlane is expected to be in alpha in v1.22
+	RootlessControlPlane = "RootlessControlPlane"
```

Comment:
This is a slight variation from the name in the KEP which was kubeamdRootlessControlPlane, I think kubeadm in the beginning does not add any value here.

Reply:
this FG name seems fine.